judges/github-events/github-events.rb

```diff
@@ -98,6 +98,14 @@ def self.comments_info(pr)
     }
   end
 
+  def self.issue_event_exists?(fact, what)
+    Fbe.fb.query(
+      "(and (eq repository #{fact.repository}) " \
```

Comment:
@tank-bohr don't forget about `where`, it should be set to `github`. We may have GitLab, for example, in the same factbase.

Reply:
@yegor256 But can we handle the events of `gitlab` in a file that called `github` -  `judges/github-events/github-events.rb` ?

Reply:
@Yegorov In the `github-events` file, we process events only from github, I think it is proposed to put a filter only on events from github like `eq where 'github'`, so as not to accidentally receive events from gitlab, for example

Reply:
@Suban05 @yegor256  ok, understood (the search was taking place in the general storage `Fbe.fb`).
Then maybe there is the same check should be done here - https://github.com/zerocracy/judges-action/pull/278